app/assets/stylesheets/base/variables.scss

```diff
@@ -111,5 +111,8 @@ $wilkinson-primary: #008ec1;
 $wilkinson-secondary: #209cc9;
 $wilkinson-tertiary: #007ba7;
 
+$law-primary: #002f7c;
+$law-secondary: #01235b;
+
```

Comment:
a little bit more about the primary vs secondary. Most schools just use our black ribbon so where the pri and sec colors really come into play are in the background colors of a few widgets that we change (theme) for each of the 10 schools such as the buttons and callouts in the right column. If you look at https://www.chapman.edu/wilkinson/art/index.aspx the primary color is the medium Cyan on the buttons and follow us and masthead. Non-schools would have red in those places. The secondary is then the hover on the buttons and tertiary the active state when you click it. As Kayla said, but as applies to widgets not the ribbon in the footer